unidist/cli/base/utils.py

```diff
@@ -0,0 +1,121 @@
+# Copyright (C) 2021 Modin authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+"""Utilities for backend runners."""
+
+import os
+import ipaddress
+import socket
+import multiprocessing as mp
+
+
+class BackendName:
+    """String representations of unidist backends."""
+
+    RAY = "ray"
+    MPI = "mpi"
+    DASK = "dask"
+    MP = "multiprocessing"
+    PY = "python"
+
+
+class Defaults:
+    """Default values for supported CLI parameters."""
+
+    EXECUTOR = "python"
+    BACKEND = BackendName.RAY
+    NUM_CPUS = "default"
+    HOSTS = "localhost"
+    REDIS_PASSWORD = "5241590000000000"
```

Comment:
Where did you take this value from?

Reply:
`5241590000000000` is default redis_password from Ray API/CLI defaults

Reply:
Can you point me to the Ray docs? Maybe we should use `secrets.token_hex(32)` as we do in Modin?

Reply:
`secrets.token_hex(32)` is incorrect default value for connecting to existing ray cluster. Correct default password can be got from public API:
```python
In [1]: from ray import ray_constants
In [2]: ray_constants.REDIS_DEFAULT_PASSWORD
Out[2]: '5241590000000000'
```

I didn't find description of this password in the docs, but I believe that public API value will be enough. What do you think? If you agree I'll change this to use Ray API to get this value.

Also, it should be fixed in Modin.

Reply:
Yes, let's use  Ray API to get the value.

If using `secrets.token_hex(32)` is incorrect, please create an issue in Modin and describe why it is wrong.

Reply:
`Default.REDIS_PASSWORD` is empty string now. Specific for Ray backend value is set in Ray runner now. It's done to divide execution for different backends (to make it works in case user installed one of dependencies sets, `pip install -e .[dask]`, for example)

Reply:
https://github.com/modin-project/modin/issues/3744 the similar Modin issue